docs/tutorials/projects/verification-system.md

```diff
@@ -0,0 +1,241 @@
+images: ["/tutorials/verification-system/disarmed.png"]
+tags: ["mlmodel", "vision", "services", "security", "camera"]
+authors: ["Sierra G."]
+languages: []
+viamresources: ["mlmodel", "vision", "camera"]
+level: "Intermediate"
+date: "28 November 2023"
+cost: "0"
+no_list: true
+# SMEs: Bijan Haney
+---
+
+Follow this tutorial to create your own machine learning (ML) enabled security system, or, "verification system."
+The verification system configuration you will use in this project combines two detectors: The first detector detects whether there are people present, thanks to a person-detection ML model you train yourself.
+You give the second detector photos of known faces and it tells you if a person is recognized.
+
+The verification system has three possible layers of detectors.
+You layer them together so that objects detected in the images your verification's system camera captures point to countdown, alarmed, or disarmed states.
+With our recommended combination of detectors, your verification system will trigger a countdown when it detects people and disarm the alarm if it detects your face with the facial detector within the countdown period.
+
+For example:
+
+{{<video webm_src="/tutorials/verification-system/demo.webm" mp4_src="/tutorials/verification-system/demo.mp4" alt="Bijan interacting with the verification system" poster="/tutorials/verification-system/trigger_1.png">}}
+
+## Requirements
+
+- A [camera](/components/camera/)
+- A computer running `viam-server` that the camera can connect to, like a [single-board computer (SBC)](/components/board/)
+
+Before configuring your camera, you must [create a robot](/fleet/machines/#add-a-new-robot).
+
+1. On the locations page of the [Viam app](https://app.viam.com), add a new robot by providing a name in the **New Robot** field and clicking **Add robot**.
+2. Click on that robot's name to go to its page.
+
+## Configure a camera
+
+Navigate to the **Config** tab of your machine's page on the [Viam app](https://app.viam.com).
+Configure the camera you want to use for your security system.
+
+We configured ours as a webcam named `"my-webcam"`.
+Follow [these instructions](/components/camera/webcam/) to do so.
+
+Connect your camera to your machine running `viam-server`, and hang it up somewhere where it can see people in your home:
+
+{{<imgproc src="/tutorials/verification-system/camera.jpeg" resize="500x" declaredimensions=true alt="Camera hanging up in office.">}}
+
+Next, configure the person detector, or, the "coarser" layer of the security system that verifies that there's a person moving.
+
+## Configure an `mlmodel` person detector
+
+Create a detector to detect when people are in your verification system camera's field of vision:
+
+1. [Capture images from your camera](/data/capture/#configure-data-capture-for-individual-components) with Viam's data management service, then [create a dataset](/data/dataset/#create-a-dataset-and-add-data).
```

Comment:
This needs more copy because it's very high level telling people what to do. 

```suggestion
To be able to detect people with your verification system, you first need to train a model that detects people and for that you need data.
You can collect this data using the data management service and your already installed camera.

(Add steps to add the data management service and configure capture, then instruct them to turn capturing on for a while while letting different people go past the camera, then instruct them to turn off data capture so that it doesn't cost them a lot of money. ALTERNATIVELY, we could instruct them to use an off the shelf person detector. I think giving both options would be nice)

1. 
you need to [Capture images from your camera](/data/capture/#configure-data-capture-for-individual-components) with Viam's data management service, then [create a dataset](/data/dataset/#create-a-dataset-and-add-data).
```

Reply:
Maybe google is just not good, but I haven't been able to find a good person detecting TF model online. Object detection, sure, but not specifically people. Vision team is making one that should be really good re. this scope https://viam.atlassian.net/browse/RSDK-5629 want me to create a ticket to track and add to this?